appveyor.yml

```diff
@@ -83,7 +83,7 @@ build_script:
 - cd %ROOT_DIRECTORY%\bindings\python
 # we use 64 bit python builds
 - if defined python (
-  b2.exe --hash openssl-version=pre1.1 warnings-as-errors=on -j2 %compiler% address-model=%model% picker-debugging=on invariant-checks=full variant=%variant% %linkflags% %include% link=shared crypto=%crypto% libtorrent-link=shared stage_module stage_dependencies
+  b2.exe -d+2 --hash openssl-version=pre1.1 warnings-as-errors=on -j2 %compiler% address-model=%model% picker-debugging=on invariant-checks=full variant=%variant% %linkflags% %include% link=shared crypto=%crypto% libtorrent-link=shared stage_module stage_dependencies
```

Comment:
oh, this looks like debugging left-overs too

Reply:
Right, I'm still on it. I did something last night to make my local MSVS compile it, and left a comment about Boost 1.67 above. Now I can't reproduce that.

Reply:
@arvidn, would you accept the workaround from the last commit? I'm about to give up on finding the real cause of the problem. Also, what to do with the warnings which are treated as errors?